fixity/fixity.py

```diff
@@ -60,10 +61,12 @@ def fetch_environment_variables(namespace):
         namespace.report_user = namespace.report_pass = None
 
 
-def scan_message(aip_uuid, status):
+def scan_message(aip_uuid, status, message):
```

Comment:
If `message` is optional, then change it to have a default value of None? `message=None`